packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -105,10 +151,27 @@ const ActivityDetailsSidebar = (props: Props) => {
   )
   const {onError, onCompleted, submitting, submitMutation, error} = useMutationProps()
   const history = useHistory()
+  const {togglePortal: toggleModal, modalPortal} = useModal({
+    id: 'createGcalEventModal'
+  })
 
-  const handleStartActivity = () => {
+  const handleStartActivity = (gcalInput?: Omit<CreateGcalEventInput, 'teamId' | 'meetingId'>) => {
     if (submitting) return
     submitMutation()
+    const handleCreateGcalEvent = (res: StartMeetingResponse) => {
+      const meetingId = getMeetingIdFromResponse(res)
+      if (!meetingId || !gcalInput) return
+      const variables = {
+        input: {
+          ...gcalInput,
+          teamId: selectedTeam.id,
+          meetingId
+        }
+      }
+      CreateGcalEventMutation(atmosphere, variables, {onError, onCompleted})
```

Comment:
Question: Is there a reason to have it as a separate mutation but not include it in start meeting mutation?

Reply:
Yeah, I think you're right! Updated 👍